.github/workflows/release-test.yml

```diff
@@ -135,6 +136,8 @@ jobs:
         fi
 
         cd Release-Specs
+        # avoid exiting on non-zero exit code
+        set +e
```

Comment:
This should not be needed with the `shell:` key above.

Reply:
It doesn't work anyway :/